### PR TITLE
Feat#37/change material ui version to mui version 5

### PR DESCRIPTION
**Pull request by MounikaReddy15**
## Purpose
_change material-ui version from 4 to mui version 5_

## Ticket number
#37

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ import * as React from 'react';
 import 'react-notifications/lib/notifications.css';
 import { hot } from 'react-hot-loader';
 import { BrowserRouter as Router, Route, Redirect } from 'react-router-dom';
-import CssBaseline from '@material-ui/core/CssBaseline';
-import { ThemeProvider } from '@material-ui/core/styles';
+import CssBaseline from '@mui/material/CssBaseline';
+import { ThemeProvider, StyledEngineProvider } from '@mui/material/styles';
```
Comment (RadhikaPPatel) [inline src/App.js:6]: Why is StyledEngineProvider needed?

**Review by RadhikaPPatel — CHANGES_REQUESTED** [review]
Get a few errors on different pages. Please handle them

image

**Comment by MounikaReddy15** [comment]
changes present in another pr #41

**Comment by MounikaReddy15** [comment]
changes present in another pr #41

**Comment by MounikaReddy15** [comment]
changes present in another pr #41

**Comment by MounikaReddy15** [comment]
changes present in another pr #41

**Pull request closed without merging**